### ISSUE TITLE
No Docker installation?

### ISSUE THREAD

**Issue by KipourosV** [hooram/ownphotos-backend#68]
Is there a way to install without using Docker?
I mean, the old traditional git clone, make install method?
My environment is LXC...

**Comment by guysoft** [comment]
Not really, mainly because we can't support all possible configurations.
You need to install
1. postgres
2. redis
3. ownphotos from this repo and its dependencies
4. ownphotos-frontend and its dependencies

The docker file lists the installation process. You will not need the ownphotos-proxy container.

It can be done, we can't support it. You are welcome to join the maintenance team

**Comment by sbrl** [comment]
Does it support SQLite3 at all @guysoft?

**Comment by guysoft** [comment]
@sbrl
SQLite3 does not support threads
Source:
https://www.sqlite.org/faq.html#q6

**Comment by sbrl** [comment]
Not sure I understand @guysoft. That link says this:

> SQLite is threadsafe.

....implying that it _does_ support threads, but by utilising a locking mechanism.

I assume that OwnPhotos requires this functionality? I'll take that as a no it doesn't then.

**Comment by guysoft** [comment]
@sbrl All my attempts to integrate something like django to use Sqlite have failed. You are welcome to try, but its beyond me. I always get thread errors eventually.

PRs are welcome

**Comment by sbrl** [comment]
@guysoft I see - thanks for the clarification.

Unfortunately, I don't really use Python all that often if I can help it, and I've never used Django before - so I'm not sure I'll be much help :confused: 

My first thoughts are that it might be a bug in the SQLite3 driver library? If SQLite3 itself claims to be threadsafe, then the driver library should be too.
